client/src/Components/Sidebar/DashboardSidebar.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import { Grid, TextField } from "@material-ui/core";
+import SearchIcon from '@material-ui/icons/Search';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import SidebarCheckbox from 'Components/Checkbox/SidebarCheckbox';
+import colors from "Components/Styles/Colors";
+
+
+const useStyles = makeStyles((theme) => ({
+    
+    main: {
+        backgroundColor : `${colors.white}`,
+        height: "100vh",
+        width: "100%",
+    },
+    textField: {
+        width: '60%',
+        marginLeft: 'auto',
+        marginRight: 'auto',            
+        paddingBottom: 0,
+        marginTop: 50,
+        fontWeight: 500,
+    },
+  }));
+
+function DashboardSidebar (props) {
```

Comment:
This is a pretty generic name, but I believe it's goal is to only be used for the Templates page right?

Also, are you using the same linter/prettier as the team? Probably something to check

Reply:
@skassam21 I intended to break this out as a generic Sidebar component that could be used in the Prospects page as well since it's essentially the same code.

Reply:
I'll check my prettier config

Reply:
ah, 'format on save' was turned off. Fixed now